packages/cli/README.md

```diff
@@ -218,6 +218,60 @@ Options:
   - `--ignore`: Ignores all of the fields on the list. Accepts all of the possible fields
   in the Web Manifest.
 
+## `fingerprint`
+
+Manages the list of fingerprints used to generate the Digital Asset Links file for the web application.
+
+Usage:
+
+```
+bubblewrap fingerprint [subcommand]
+``` 
+
+Global options:
+  - `--manifest=<manifest>`: path to the Trusted Web Activity configuration.',
+
+### Subcommands:
+
+#### `add`
+Adds a fingerprint to the project configuration.
+
+Usage:
+
+```
+bubblewrap fingerprint add [SHA-256 fingerprint] <flags>
+```
+
+Flags:
+ - `--name=<name>`: he name for the fingerprint.
```

Comment:
Could you add something saying what the `name` does?

Reply:
updated.